protected/humhub/modules/user/controllers/AuthController.php

```diff
@@ -211,6 +211,36 @@ public function onAuthSuccess(BaseClient $authClient)
         return $this->redirect(['/user/registration']);
     }
 
+    /**
+     * Do log in user
+     *
+     * @param $user
+     * @param $authClient
+     * @param $redirectUrl
+     * @param $isOAuth
+     * @return array
+     */
+    protected function doLogin($user, $authClient, $redirectUrl, $isOAuth = false)
```

Comment:
Make we could mark the visibility of this method to `private`? 

Also the name of the param `$isOAuth` is a bit misleading. If I understand correctly, this is about all non "local" auth methods? (e.g. LDAP, OAuth2, SAML, ...)

Reply:
Yes, I will think then how to do it better.